kratos/geometries/line_2d_3.h

```diff
@@ -373,21 +378,89 @@ class Line2D3 : public Geometry<TPointType>
      * @return True if the point is inside, false otherwise
      */
     bool IsInside(
-        const CoordinatesArrayType& rPoint, 
+        const CoordinatesArrayType& rPoint,
         CoordinatesArrayType& rResult,
-        const double Tolerance = std::numeric_limits<double>::epsilon() 
+        const double Tolerance = std::numeric_limits<double>::epsilon()
         ) override
     {
-        this->PointLocalCoordinates( rResult, rPoint );
+        PointLocalCoordinates( rResult, rPoint );
 
-        if ( std::abs( rResult[0] ) <= (1.0 + Tolerance) )
-        {
+        if ( std::abs( rResult[0] ) <= (1.0 + Tolerance) ) {
```

Comment:
this check does not seem to be correct

Reply:
It is correct, the number of local coordinates is 1, between -1 and 1, it is exactly the same as the line with two nodes

Reply:
https://github.com/KratosMultiphysics/Kratos/blob/310b2f21ac91bbeb71aef35bdaad8c6a229d7850/kratos/geometries/line_2d_2.h#L888

Reply:
sorry, my bad, now I understood, nevermind